pysindy/pysindy.py

```diff
@@ -157,12 +170,15 @@ def __init__(
         feature_names=None,
         t_default=1,
         discrete_time=False,
+        inputs_per_library=None,
     ):
         if optimizer is None:
             optimizer = STLSQ()
         self.optimizer = optimizer
         if feature_library is None:
             feature_library = PolynomialLibrary()
+        if inputs_per_library is None:
```

Comment:
Is this a typo? It looks like `self.inputs_per_library` will only be initialized if `inputs_per_library` is None.

Reply:
yes, inputs_per_library I originally tried to implement in the pysindy.py file, but then switched to using it directly in the GeneralizedLibrary object. So this is defunct code, and I removed it in my recent commit